test/unit/test_model.py

```diff
@@ -440,7 +440,7 @@ def test_set_scsi_slot(self):
     def test_set_drive_page_file_exist(self):
         file_name = "/tmp/an_avaiable_page_file.bin"
         os.system("touch {0}".format(file_name))
-        ps = r"-device \S+page_file={0}[\s,]".format(file_name)
+        ps = r"-device \S+page_file={0}[\s,]?".format(file_name)
```

Comment:
so why should we check the [\s,] here if they might not exist?

Reply:
Good question...